src/test/java/core/basesyntax/service/ParserServiceTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceTest {
```

Comment:
Think about it everywhere
```suggestion
class ParserServiceImplTest {
```